cypress/integration/rich-text/RichTextEditor.spec.ts

```diff
@@ -430,6 +430,28 @@ describe('Rich Text Editor', { viewportHeight: 2000 }, () => {
           ).to.have.length(1);
         });
       });
+
+      //todo test should not delete adjacent non-command text
+
+      it('should not delete adjacent text', () => {
+        richText.editor.click().type('test/{downarrow}{enter}{enter}');
+        richText.expectSnapshotValue();
+      });
+
+      it('should work inside headings', () => {
+        richText.editor.click().type('Heading 1');
+        richText.toolbar.toggleHeading(BLOCKS.HEADING_1);
+        richText.editor.click().type('/{enter}{enter}');
+
+        //this is used instead of snapshot value because we have randomized entry IDs
```

Comment:
This is getting more and more annoying. I will see if we can make these random ids more predictable (e.g. counter)